docs/release-notes/8.0.35-27.md

```diff
@@ -0,0 +1,68 @@
+# Percona Server for MySQL 8.0.35-27 (2023-12-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL](../quickstart-overview.md).
+
+[Percona Server for MySQL 8.0.35-27](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.35 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-35.html) in addition to enterprise-grade features developed by Percona.
+
+## Release highlights
+
+Percona Server for MySQL implements telemetry that fills in the gaps in our understanding of how you use Percona Server to improve our products. Participation in the anonymous program is optional. You can opt-out if you prefer not to share this information. Find more information in the [Telemetry on Percona Server for MySQL](../telemetry.md) document.
+
+This merge fixes the following issue:
+
+* [PS-8849](https://jira.percona.com/browse/PS-8849): The server exited with the `Assertion failure: row0mysql.cc:218:len < 256 * 256 thread`.
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.35 and included in Percona Server for MySQL are the following:
+
+* Upgraded the linked OpenSSL library to OpenSSL 3.0.10.
+* Removed the printed query string limit to display the characters for a detected deadlock section of the engine status log.
+* Fixed a procession of single-character tokens by an FTS parser plugin.
+
+### Deprecations and removals
```

Comment:
Maybe call this section 
Deprecations
I didn't see any removals.

Reply:
fixed